src/main/modules/api/cos-api.ts

```diff
@@ -0,0 +1,19 @@
+import { AxiosInstance } from 'axios';
+import { LoggerInstance } from 'winston';
+
+export class COSApi {
```

Comment:
I think the filename and class name should be CosApi. It's the convention I normally but you don't have to

Reply:
initially i wrote that but i thought as COS is an acronym, it might be best to name it like that but i understand. Ill change it.